exampleCourse/questions/element/matching/question.html

```diff
@@ -0,0 +1,113 @@
+<pl-question-panel>
+    <p>
+        This question illustrates different use cases for the
+        <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-matching-element"><code>pl-matching</code></a>
+        element.
+        </p>
+    <p>
```

Comment:
For separating the examples, can you use boxes (cards) like those in `elements/integerInput`?